treatment-service/services/poller_service.go

```diff
@@ -1,4 +1,4 @@
-package server
+package services
```

Comment:
Refactoring of the `Poller` object of the `server` package into a `PollerService` of the `services` package that gets initialised and started up in both the XP Treatment Service and XP plugin runner.